doc/userDocs/usersGuide.md

```diff
@@ -788,6 +905,73 @@ The `kubeconfigContext` parameter specifies the name of the context for your clu
 
 where `kubernetes-admin@kubernetes` is the name of your cluster's context. 
 
+##### kubeconfigContext
+
+The `kubeconfigContext` parameter specifies the name of the context for your cluster such as `kubernetes-admin@kubernetes`. This can be found in your *kubeconfig* file. Set an empty string (`""`) to use the `current-context` defined in the `kubeconfigFile`.
+
+| Configuration Parameter: kubeconfigContext             |
+| ------------------------------------------------------ |
+| `"kubeconfigContext" : "kubernetes-admin@kubernetes",` |
+
+where `kubernetes-admin@kubernetes` is the name of your cluster's context. 
+
+##### namespacePrefix
+
+The `namespacePrefix` parameter specifies the prefix to be used in the generation of 
+the namespace names used by Weathervane on this Kubernetes cluster. See [Configuring 
+  Namespaces](#namespaces) for more detail.
+  
+
+| Configuration Parameter: namespacePrefix             |
+| ------------------------------------------------------ |
+| `"namespacePrefix" : "auction",` |
+
+where you can replace *auction* with your desired prefix. The default value is _auction_.
+
+##### namespaceSuffix
+
+The `namespaceSuffix` parameter specifies the suffix to be used in the generation of 
+the namespace names used by Weathervane on this Kubernetes cluster. See [Configuring 
+  Namespaces](#namespaces) for more detail.
+  
+
+| Configuration Parameter: namespaceSuffix             |
+| ------------------------------------------------------ |
+| `"namespaceSuffix" : "",` |
+
+The default value is the empty string. 
+
+##### namespaces
+
+The `namespaces` parameter specifies a list of namespace names to used by Weathervane 
+on this Kubernetes cluster. The value specified for this parameter must be a JSON list 
+of strings, each of which is a namespace name.  If a non-empty list is specified for 
+this parameter, then Weathervane will not generate namespace names for this cluster. 
+See [Configuring Namespaces](#namespaces) for more detail.
+  
+
+| Configuration Parameter: namespaceSuffix             |
+| ------------------------------------------------------ |
+| `"namespaceSuffix" : "",` |
```

Comment:
Should be "namespaces"x2 here.
Also, is there a note anywhere that "namespaces" takes precedence over Prefix and Suffix?

Reply:
I fixed the copy/paste error.

The text stating 

> If a non-empty list is specified for this parameter, then Weathervane will not generate namespace names for this cluster.

was supposed to indicate that the prefix and suffix would not be used, but I have made this more explicit.